### PR TITLE
MINOR: Improved configurations formatting

### PR DESCRIPTION
**Pull request by mimaison**
Fixed a few formatting issues in the configurations

### Committer Checklist (excluded from commit message)
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation (including upgrade notes)

### CODE REVIEW

File: clients/src/main/java/org/apache/kafka/clients/producer/ProducerConfig.java [hunk 1]
```diff
@@ -113,8 +113,8 @@
             + "for retriable send failures. The producer may report failure to send a record earlier than this config if "
             + "either an unrecoverable error is encountered, the retries have been exhausted, "
             + "or the record is added to a batch which reached an earlier delivery expiration deadline."
-            + "The value of this config should be greater than or equal to the sum of " + REQUEST_TIMEOUT_MS_CONFIG
-            + " and " + LINGER_MS_CONFIG + ". ";
+            + "The value of this config should be greater than or equal to the sum of <code>" + REQUEST_TIMEOUT_MS_CONFIG + "/<code> "
```
Comment (hachikuji) [inline clients/src/main/java/org/apache/kafka/clients/producer/ProducerConfig.java]: Typo: `/<code>`

File: clients/src/main/java/org/apache/kafka/clients/producer/ProducerConfig.java [hunk 2]
```diff
@@ -190,8 +190,8 @@
             + " ordering of records because if two batches are sent to a single partition, and the first fails and is retried but the second"
             + " succeeds, then the records in the second batch may appear first. Note additionall that produce requests will be"
             + " failed before the number of retries has been exhausted if the timeout configured by"
-            + " " + DELIVERY_TIMEOUT_MS_CONFIG + " expires first before successful acknowledgement. Users should generally"
-            + " prefer to leave this config unset and instead use " + DELIVERY_TIMEOUT_MS_CONFIG + " to control"
+            + " <code>" + DELIVERY_TIMEOUT_MS_CONFIG + "</code> expires first before successful acknowledgement. Users should generally"
```
Comment (hachikuji) [inline clients/src/main/java/org/apache/kafka/clients/producer/ProducerConfig.java:193]: Do we need a space after "by"?

**Review by hachikuji — COMMENTED** [review]
Thanks for the cleanup, just a couple nits.

File: clients/src/main/java/org/apache/kafka/clients/producer/ProducerConfig.java (hunk 2 above)
Comment (mimaison) [reply clients/src/main/java/org/apache/kafka/clients/producer/ProducerConfig.java:193]: There's a leading space on the next line, before `<code>`.

**Comment by hachikuji** [comment]
retest this please

File: clients/src/main/java/org/apache/kafka/clients/producer/ProducerConfig.java (hunk 2 above)
Comment (hachikuji) [reply clients/src/main/java/org/apache/kafka/clients/producer/ProducerConfig.java:193]: Ah, you're right. Not sure how I missed that.

**Review by hachikuji — APPROVED** [review]
LGTM. Thanks for the patch!

**Pull request merged**